changelog.d/13650.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug where redactions were not being sent over federation if we did not have the original event.
```

Comment:
Maybe `Fix a long-standing bug`? (I'm guessing this didn't regress in any particular release.)